controllers/suite_test.go

```diff
@@ -129,6 +131,12 @@ var _ = BeforeSuite(func(done Done) {
 	cUtil = controllerUtil.New(ctx, k8sClient, cr)
 	Expect(util).ToNot(BeNil())
 
+	// Generation of 3 char long random string
+	cusKey = cUtil.RandSeqString(3)
+
+	mockData.CustomerTestProjectInput.Spec.Name += cusKey
+	mockData.CustomerTestProjectInput.Spec.Key = strings.ToUpper(cusKey)
+
 	_ = util.CreateProject(mockData.CustomerTestProjectInput, ns)
```

Comment:
This project created here is not deleted in AfterSuite.

Reply:
Ack